frameworks/autoxgboost/setup.sh

```diff
@@ -2,10 +2,25 @@
 HERE=$(dirname "$0")
 VERSION=${1:-"stable"}
 REPO=${2:-"ja-thomas/autoxgboost"}
-# currently both stable and latest maps to master branch
+
+# Version can be specified as 'stable', 'latest', a branch or a commit hash (indicated by starting with '#')
 if [[ "$VERSION" == "latest" || "$VERSION" == "stable" ]]; then
-    VERSION="master"
+  VERSION="master"
+fi
+
+if [[ "$VERSION" =~ ^# ]]; then
+  VERSION="${VERSION:1:8}"
```

Comment:
```suggestion
  VERSION="${VERSION:1}"
```
well, my point was not to use only 7 chars if user is providing more, I just didn't want to force the user to provide the full hash when the "default" 7 chars is almost always enough.

Reply:
Gotcha. I thought you also weren't satisfied by how noisy the framework version looks with 40 character hash.

Reply:
Nvm, I see you addressed that when writing to the results file.